README.md

```diff
@@ -1,2 +1,62 @@
 # inverted-index
 Inverted index data structure built with JavaScript
+
+[![Build Status](https://travis-ci.org/andela-korjiewuru/inverted-index.svg?branch=development)](https://travis-ci.org/andela-korjiewuru/inverted-index)
+[![Code Climate](https://codeclimate.com/github/andela-korjiewuru/inverted-index/badges/gpa.svg)](https://codeclimate.com/github/andela-korjiewuru/inverted-index)
+[![Coverage Status](https://coveralls.io/repos/github/andela-korjiewuru/inverted-index/badge.svg?branch=development)](https://coveralls.io/github/andela-korjiewuru/inverted-index?branch=development)
+
+### dexIt Inverted Index Application
+
+dexIt enables you to upload your JSON array files, create an index map of words in to the corresponding JSON document in the array, and perform search on the words present.
+
+#### Using the App
```

Comment:
Also, there's no link to the hosted application where a user can access and use the application.